app/core/DeeplinkManager/Handlers/handleEthereumUrl.test.ts

```diff
@@ -271,4 +284,28 @@ describe('handleEthereumUrl', () => {
       expect.any(Object), // The exact expectations here depend on the intended behavior
     );
   });
+
+  it('should switch to mainnet when isEvmSelected is false', async () => {
```

Comment:
nit: wording 
https://github.com/MetaMask/contributor-docs/blob/main/docs/testing/unit-testing.md#use-it-to-specify-the-desired-behavior-for-the-code-under-test

```suggestion
  it('switch to mainnet when isEvmSelected is false', async () => {
```